src/grammar/block.rs

```diff
@@ -63,6 +64,57 @@ pub(super) fn parse_stmt(p: &mut Parser) {
     p.finish();
 }
 
+fn parse_execute_immediate_stmt(p: &mut Parser) {
+    p.start(SyntaxKind::ExecuteImmediateStmt);
+    p.expect(T![execute]);
+    p.expect(T![immediate]);
+    // Parse String
+    if !p.eat(T![unquoted_ident]) {
+        p.expect(T![quoted_literal]);
+    }
+    // handle INTO, USING and RETURN/RETURNING clauses
+    if p.at(T![into]) {
+        parse_into_clause(p, true);
+    }
```

Comment:
Seems we're missing the [`bulk_collect_into_clause`](https://docs.oracle.com/en/database/oracle/oracle-database/19/lnpls/RETURNING-INTO-clause.html#GUID-38F735B9-1100-45AF-AE71-18FB74A899BE__CJAIAGHJ)

Reply:
True